### PR TITLE
add sentieon specific docs

### PR DESCRIPTION
**Pull request by MaxUlysse**
# nf-core/sarek pull request

Many thanks for contributing to nf-core/sarek!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Piellorieppe is one of the main massif in the Sarek National Park.
 - [#164](https://github.com/nf-core/sarek/pull/164) - Add `--no_gatk_spark` params and tests
 - [#167](https://github.com/nf-core/sarek/pull/167) - Add `--markdup_java_options` documentation
 - [#169](https://github.com/nf-core/sarek/pull/169) - Add `RELEASE_CHECKLIST.md` document
+- [#169](https://github.com/nf-core/sarek/pull/169) - Add `Sentieon` documentation
```
Comment (ggabernet) [inline CHANGELOG.md:24]:
```suggestion
- [#175](https://github.com/nf-core/sarek/pull/175) - Add `Sentieon` documentation
```

File: docs/sentieon.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+> Improved accuracy and genome characterization.
+> Machine learning enhanced filtering producing top variant calling accuracy.
+
+This tool is enabled within Sarek if `--sention` is specified and if `--tools DNAscope` is specified cf [--tools](#--tools).
+
+### Somatic SNV/INDEL Variant Calling - TNscope
+
+> Winner of ICGC-TCGA DREAM challenge.
+> Improved accuracy, machine learning enhanced filtering.
+> Supports molecular barcodes and unique molecular identifies.
```
Comment (ggabernet) [inline docs/sentieon.md:53]:
```suggestion
> Supports molecular barcodes and unique molecular identifiers.
```

File: docs/sentieon.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Sentieon Analysis Pipelines & Tools
+
+The following Sentieon Analysis Pipelines & Tools are available within Sarek.
+
+### Alignment
+
+> Sentieon BWA matches BWA-MEM with > 2X speedup.
+
+This tool is enabled by default within Sarek if `--sention` is specified and if the pipeline is started with the `mapping` [step](usage.md#--step).
```
Comment (ggabernet) [inline docs/sentieon.md:32]:
```suggestion
This tool is enabled by default within Sarek if `--sentieon` is specified and if the pipeline is started with the `mapping` [step](usage.md#--step).
```

File: docs/sentieon.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+### Germline SNV/INDEL Variant Calling - DNAseq
+
+> Precision FDA award-winning software.
+> Matches GATK 3.3-4.1, and without down-sampling.
+> Results up to 10x faster and 100% consistent every time.
+
+This tool is enabled within Sarek if `--sention` is specified and if `--tools DNAseq` is specified cf [--tools](#--tools).
```
Comment (ggabernet) [inline docs/sentieon.md:40]:
```suggestion
This tool is enabled within Sarek if `--sentieon` is specified and if `--tools DNAseq` is specified cf [--tools](#--tools).
```

File: docs/sentieon.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+### Germline SNV/INDEL Variant Calling - DNAscope
+
+> Improved accuracy and genome characterization.
+> Machine learning enhanced filtering producing top variant calling accuracy.
+
+This tool is enabled within Sarek if `--sention` is specified and if `--tools DNAscope` is specified cf [--tools](#--tools).
```
Comment (ggabernet) [inline docs/sentieon.md:47]:
```suggestion
This tool is enabled within Sarek if `--sentieon` is specified and if `--tools DNAscope` is specified cf [--tools](#--tools).
```

File: docs/sentieon.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+> Improved accuracy, machine learning enhanced filtering.
+> Supports molecular barcodes and unique molecular identifies.
+
+This tool is enabled within Sarek if `--sention` is specified and if `--tools TNscope` is specified cf [--tools](#--tools).
+
+### Structural Variant Calling
+
+> Germline and somatic SV calling, including translocations, inversions, duplications and large INDELs
+
+This tool is enabled within Sarek if `--sention` is specified and if `--tools DNAscope` is specified cf [--tools](#--tools).
```
Comment (ggabernet) [inline docs/sentieon.md:61]:
```suggestion
This tool is enabled within Sarek if `--sentieon` is specified and if `--tools DNAscope` is specified cf [--tools](#--tools).
```

File: docs/usage.md [hunk 7]
```diff
@@ -336,9 +336,7 @@ Available: `ASCAT`, `ControlFREEC`, `FreeBayes`, `HaplotypeCaller`, `Manta`, `mp
 If [Sentieon](https://www.sentieon.com/) is available, use this to enable it for preprocessing, and variant calling.
 Adds the following tools for the [`--tools`](#--tools) options: `DNAseq`, `DNAscope` and `TNscope`.
 
-Please refer to the [nf-core/configs](https://github.com/nf-core/configs#adding-a-new-pipeline-specific-config) repository on how to make a pipeline-specific configuration file based on the [munin-sarek specific configuration file](https://github.com/nf-core/configs/blob/master/conf/pipeline/sarek/munin.config).
-
-Or ask us on the [nf-core Slack](http://nf-co.re/join/slack) on the following channels: [#sarek](https://nfcore.slack.com/channels/sarek) [#configs](https://nfcore.slack.com/channels/configs).
+More information in the [sention](sentieon.md) documentation.
```
Comment (ggabernet) [inline docs/usage.md:339]:
```suggestion
More information in the [sentieon](sentieon.md) documentation.
```

File: docs/sentieon.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+### Somatic SNV/INDEL Variant Calling - TNscope
+
+> Winner of ICGC-TCGA DREAM challenge.
+> Improved accuracy, machine learning enhanced filtering.
+> Supports molecular barcodes and unique molecular identifies.
+
+This tool is enabled within Sarek if `--sention` is specified and if `--tools TNscope` is specified cf [--tools](#--tools).
```
Comment (ggabernet) [inline docs/sentieon.md:55]:
```suggestion
This tool is enabled within Sarek if `--sentieon` is specified and if `--tools TNscope` is specified cf [--tools](#--tools).
```

**Comment by ggabernet** [comment]
PS: I hate the spelling of `sentieon` too...

**Comment by MaxUlysse** [comment]
> PS: I hate the spelling of `sentieon` too...

Hate it sooo muuuuch

**Pull request merged**